### DISCUSSION TITLE
Xapp-sn-watcher: Block specific tray icons

### DISCUSSION THREAD

**Discussion by SpoOokY1983** [linuxmint/discussions#547] [Questions]
Hi there,

is it possible to somehow block specific tray icons from being shown?
I am using Mint 22 Cinnamon and legacy tray icons of game launchers such as BattleNet and RSI-Launcher (Star-Citizen) shown in the tray break VRR until a restart of the system or kill of the process xapp-sn-watcher. It seems like this is actually a well known gnome shell bug which apparently never got fixed. 
To manually fix it I start e.g. the RSI-Launcher, start Star-Citizen, quickly shutdown the launcher and pkill xapp-sn-watcher. But that is a pretty annoying way. It would much rather be great if I could somehow prevent the tray icons from being shown. 
Cinnamon has no option for that, Wine too. Hence the question is if I can somehow tell the process to not show those icons. 

Thank you!

**Comment by 100savage** [comment]
You could write a bash script to do what you are calling the manually way. Then set up that bash script to start up when you log into your system. It may take a little time, but you can create your own fix for this.

**Reply by SpoOokY1983** [reply]
Yeah, sure, but we are talking about Mint. The go-to-distro for all Linux newcomers and people just wanting something that works without diving too much into the system. This bug breaks a fundamental function for gaming, one advertised key element, and I would say this potentially drives new Linux users back to windows.
